packages/kdl/kdl.0.2.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "OCaml implementation of the KDL Document Language"
+maintainer: "eilvelia <hi@eilvelia.cat>"
+authors: "eilvelia <hi@eilvelia.cat>"
+license: "MPL-2.0"
+tags: ["data-serialization-format" "configuration-format"]
```

Comment:
Consider adding a tag `org:eilvelia` which will act like a namespace for your packages, eg https://ocaml.org/packages/search?q=tag:"org:erratique"

Reply:
I'm not sure it's needed (most uses on opam seem to be for organizations/projects and not users, and there's also "author:"), but added!